### PR TITLE
login form added

### PR DESCRIPTION
**Pull request by agrim123**
Added a login screen in accordance to issue #10 . a live preview is available on https://slack12345.herokuapp.com (this is my instance of jinora for testing purpose)

### CODE REVIEW

**Comment by tnmy44** [comment]
Nice work! Can you make few changes, though:
- Set focus on the text box by default when the page is opened.
- User should be able to press the enter key in place of clicking on the "Start Chatting" button.
- If the user enters an empty string, assign a random name from the defaultNames array.

**Comment by karandesai-96** [comment]
I haven't seen the diffs till now, but it really looks good already ! :smile:

**Comment by karandesai-96** [comment]
Can this be merged ?

**Comment by ketanhwr** [comment]
Damn this looks nice. :+1:

File: public/style.css [hunk 1]
```diff
@@ -1,34 +1,41 @@
+@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300);
```
Comment (csoni111) [inline public/style.css]: It gives the following warning ` The page at 'https://chat.sdslabs.co/' was loaded over HTTPS, but requested an insecure stylesheet 'http://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300'. This content should also be served over HTTPS.`

**Comment by csoni111** [comment]
Another issue is that, if you ban a nick from slack. It still gets accepted on the login screen, but only when the user tries to send a message with the banned nick. He gets that old prompt box asking for another username.

File: public/style.css (hunk 1 above)
Comment (uttu357) [inline public/style.css]: Fixed with a7ddfba.

**Comment by uttu357** [comment]
@csoni111 It would be nice if you could file a separate issue for this. :)

**Comment by rishidvgn** [comment]
Hi people,
I am a former member of sdslabs, but due to some reason this thread still
has me in the mailing list. Please append the group and list.

Warm regards,
Rishi Devgan

**Comment by rishidvgn** [comment]
Never mind, I just unsubscribed myself.

File: public/style.css (hunk 1 above)
Comment (captn3m0) [inline public/style.css]: Always use https: https://www.paulirish.com/2010/the-protocol-relative-url/

Comment (DhavalKapil) [inline public/style.css]: My bad, This article make a lot of sense.

**Pull request merged**